packages/service-library/src/servicelib/aiohttp/tracing.py

```diff
@@ -91,6 +99,10 @@ def setup_tracing(
 
     # Instrument aiohttp client
     AioHttpClientInstrumentor().instrument()
-    if instrument_aiopg:
+    if HAS_AIOPG:
         AiopgInstrumentor().instrument()
+        _logger.info("Added aiopg opentelemetry instrumentation.")
+    if HAS_BOTOCORE:
+        BotocoreInstrumentor().instrument()
+        _logger.info("Attempting to add botocore opentelemetry autoinstrumentation...")
```

Comment:
Attempted? it is already instrumented, right?
TIP: i would have done it with a general template message that logs something like "intrumentation of {tool_name} {action}" and `action` can be `{"done", "skipped"}` OR using the context logger as well ...

Reply:
thx